src/test/java/org/fundacionjala/coding/christian/movies/CustomerTest.java

```diff
@@ -0,0 +1,97 @@
+package org.fundacionjala.coding.christian.movies;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Developer by Christian.
+ */
+public class CustomerTest {
```

Comment:
@Chritian92 
Try to have unit test for Children, NewRelease and Regular classes.